packages/vue-instantsearch/src/__tests__/common-widgets.test.js

```diff
@@ -362,6 +362,45 @@ const testSetups = {
       document.body.appendChild(document.createElement('div'))
     );
 
+    await nextTick();
+  },
+  async createCurrentRefinementsWidgetTests({
+    instantSearchOptions,
+    widgetParams,
+    testParams,
+  }) {
+    const refinementListAttributes = Object.keys(
+      instantSearchOptions.initialUiState?.indexName.refinementList || {}
+    );
+
+    const search = (h) =>
+      h(AisInstantSearch, { props: instantSearchOptions }, [
+        ...refinementListAttributes.map((attribute) =>
+          h(AisRefinementList, { props: { attribute } })
+        ),
+        h(AisCurrentRefinements, { props: widgetParams }),
+        h(GlobalErrorSwallower),
+      ]);
+
+    mountApp(
+      {
+        render: renderCompat((h) =>
+          testParams?.formWrapperSubmitHandler
```

Comment:
it would be good if the submit handler was defined for every test in a widget (the others can just have it as a noop)